src/BloomExe/MiscUI/ProblemReporterDialog.cs

```diff
@@ -148,9 +150,22 @@ private void UpdateDisplay(object sender, EventArgs e)
 #endif
 		}
 
+		public bool IsLegalEmail(string emailAddress)
```

Comment:
We have something similar IIRC in the (C#) dialog for signing up for bloom library. Could we share?